src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -29,6 +30,31 @@ const containerStyles = emotion.css`
     ${getZIndex('banner')}
 `;
 
+const measureTiming = (name: string) => {
+    const perf = window.performance;
+    const startKey = `${name}-start`;
+    const endKey = `${name}-end`;
+
+    const start = () => {
+        perf.mark(startKey);
+    };
+
+    const endAndReturn = () => {
+        perf.mark(endKey);
+        perf.measure(name, startKey, endKey);
+        const measureEntries = perf.getEntriesByName(name, "measure");
+        const timeTakenFloat = measureEntries[0].duration;
+        const timeTakenInt = Math.trunc(timeTakenFloat);
```

Comment:
It probably doesn't really matter, but would [Math.round](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round) make more sense here?

Reply:
Yep, definitely a better suggestion.